service/authentication/AuthenticationEvents.ts

```diff
@@ -7,7 +7,12 @@ const AuthenticationEvents = {
      * userIdentity - if user has been logged in then it contains user name. If
      *                contains 'null' or 'undefined' then user is not logged in.
      */
-    IDENTITY_UPDATED: 'authentication.identity_updated'
+    IDENTITY_UPDATED = 'authentication.identity_updated'
 };
 
-module.exports = AuthenticationEvents;
+export const IDENTITY_UPDATED = AuthenticationEvents.IDENTITY_UPDATED;
+
+// TODO: this was a pre-ES6 module using module.exports = AuthenticationEvents which doesn't translate well
```

Comment:
Can you pl remove the TODO now?

Reply:
Ah - I'd left that one in place so that I could have parallel PRs due to all the merge conflicts I was creating.

Either merge this in as it is as I intend to go back over all of the files in a later PR.

or

Let's get all of the RTC related PRs merged (one at a time) before we attempt to do the others